wlauto/instrumentation/freqsweep/__init__.py

```diff
@@ -33,6 +33,9 @@ class FreqSweep(Instrument):
         - Setting the runner to 'by_spec' increases the chance of successfully
           completing an agenda without encountering hotplug issues
         - If possible disable dynamic hotplug on the target device
+        - This instrument does not automatically taskset workloads to the cores
+          being swept, it is most likely that this is behaviour you want. To
```

Comment:
This is ambiguous -- suggests that *not* tasksetting is the "behaviour you want"; I assume the oposite is meant. Should be rephrased.